app/javascript/components/layouts/CardLayout.js

```diff
@@ -12,7 +12,6 @@ const CardLayout = ({ children, pageHeader, tabSection, toolbar }) => {
     document.body.classList.add('bg-snow')
   }, [])
 
-  if (tabSection && !tabSection.currentUrl) { tabSection.currentUrl = window.location.pathname }
```

Comment:
What did this line accomplish or not accomplish? Just trying to clarify

Reply:
this was selecting the active tab based on the current url, so for example if you're on /supplementals  have the supp app tab highlighted as active, and if you're on the plain application page, have the application page highlighted.

I had to update it because I added a param to the application url that has ?leaseup=true on it, so this line wasn't working. I had a different line of code that had the full url, then split it on the hostname, but I thought it made more sense to just explicitly set which tab was active wherever you're specifying the tabs.